### PR TITLE
New Board

### PR DESCRIPTION
**Pull request by Xorima**
## Description

Grants admin rights to the new board members

### Issues Resolved

[List any existing issues this PR resolves]

### Check List
- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by Xorima** [comment]
For some reason it wants to Delete and then create our memberships again?

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```
Comment (damacus) [inline .gitattributes]: Do we want to add an option to make sure there's an ending new line too?

Comment (Xorima) [reply .gitattributes]: is there a way to do this?

Comment (damacus) [reply .gitattributes]: Can't find anything afters a quick Google

**Pull request merged**